<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bmInfoWidget</class>
 <widget class="QWidget" name="bmInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>131</width>
     <height>301</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>81</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>8</number>
    </property>
    <property name="verticalSpacing">
     <number>16</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_mt1">
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_mt2">
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_mt3">
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_mt4">
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_mt5">
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_mt6">
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="label_mt7">
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="label_mt8">
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_mt9">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_mt10">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_mt11">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_mt12">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_mt13">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="label_mt14">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QLabel" name="label_mt15">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QLabel" name="label_mt16">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_mt17">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_mt18">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_mt19">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_mt20">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_mt21">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="label_mt22">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QLabel" name="label_mt23">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QLabel" name="label_mt24">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_mt25">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_mt26">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_mt27">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_mt28">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="label_mt29">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLabel" name="label_mt30">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QLabel" name="label_mt31">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QLabel" name="label_mt33">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_mt33_2">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_mt34">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_mt35">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_mt36">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLabel" name="label_mt37">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLabel" name="label_mt38">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QLabel" name="label_mt39">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="4" column="7">
     <widget class="QLabel" name="label_40">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_mt41">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_mt42">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_mt43">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_mt44">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="label_mt45">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QLabel" name="label_mt46">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QLabel" name="label_mt47">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QLabel" name="label_mt48">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>11</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>236</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <widget class="QLabel" name="label_actual">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../shaper/images.qrc">:/status/images/icon_grn.ico</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_last">
      <property name="text">
       <string>12:00:00</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_version">
      <property name="text">
       <string>v.1.0.7.34</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_errors">
      <property name="text">
       <string>Ошб:0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>57</y>
     <width>58</width>
     <height>224</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_ready">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Готов</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_COM3">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>COM3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_COM4">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>COM4</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_ATU">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>АТУ</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_Key">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>Ключ</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_Out">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_Test">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>Тест</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_Timer">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>Таймер</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_Mem">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>Память</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_PNP">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>PtoP</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_consol">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string>Консоль</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_OTU">
      <property name="text">
       <string>О</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLabel" name="label_30">
      <property name="text">
       <string>БРОК</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../shaper/images.qrc"/>
 </resources>
 <connections/>
</ui>
